### DISCUSSION TITLE
React, Hono, then deploy it on serverless CF Workers

### DISCUSSION THREAD

**Discussion by enriicsolaa** [vikejs/vike#1647] [Discussion]
Well, this is my question. I need to start a project with the frontend made with React and SSR, using filesystem routing, the backend and APIs made with Hono, and be able to deploy it serverless on Cloudflare Workers.

Would this be possible? I haven't found anything about it in the documentation, nor a similar example.

Thank you very much :)

**Comment by brillout** [comment]
Regarding Hono, @phonzammi recently added it to Bati.

As for Cloudlfare Workers, yes it's possible, see Vike's docs and previous GitHub Discussions. Once Vite ships its new Vite Ecosystem API, you'll then even be able to dev directly against workerd.

Contribution welcome to create a Vike + Hono + Cloudflare Workers example.

**Comment by enriicsolaa** [comment]
I tried to do this. This is my error. The page shows this:
![image](https://github.com/vikejs/vike/assets/56979704/635ec85c-c7a3-4150-814c-ee49615e5787)

And the console shows this:
![image](https://github.com/vikejs/vike/assets/56979704/efce4921-b996-4121-8b08-8f815cc53c01)

**Comment by enriicsolaa** [comment]
The NOSENTRY error from the console is fixed downgrading to 3.55.0. The random text from the browser stills.

**Reply by brillout** [reply]
I wouldn't know how this could be related to Vike. But reproduction welcome that showcases the error upon upgrading from `3.55.0` to `latest`, I'll have a look at it then.

**Reply by enriicsolaa** [reply]
> I wouldn't know how this could be related to Vike. But reproduction welcome that showcases the error upon upgrading from `3.55.0` to `latest`, I'll have a look at it then.

you can se the pictures?
Any idea for the random text in the first picture?
